package/MDAnalysis/coordinates/DCD.py

```diff
@@ -191,11 +191,10 @@ def _read_next_timestep(self, ts=None):
         if ts is None:
             # use a copy to avoid that ts always points to the same reference
             # removing this breaks lammps reader
-            ts = self.ts.copy()
+            ts = self.ts # why is this copy required ??
```

Comment:
I don’t understand the comment — remove?

Reply:
This copy used to be required for the lammps reader 5 years ago as the comment says. This might have changed due to refactorings in the lammps reader since then.

However, the changes you had to make in the tests mean reaming the copy will break user code. Pretty sure a good number of phd students using only dcd files never noticed that `ts.positions` is only a reference. Maybe think if it is worth it to keep the copy for compatibility.

Reply:
@kain88-de i agree that this is a dangerously non-backwards compatible change. We can go with a deprecate warning here and then move to 3.0? What do people think @MDAnalysis/coredevs?

Reply:
If it's a 30% performance increase and if it hasn't actually been documented then I am actually tending towards getting it fixed sooner than later. Maybe deprecate in 2.4 and change in 2.5 if there are strong feelings about its impact.

Reply:
If it's likely to affect user behaviour then imho coordinates is one of the places we want to be the most cautious with. DCD is an unnaturally popular format, my 2 cents is that I'd prefer bump up 3.0's timeline than make a major break here.

Reply:
I'll move to 3.0

Reply:
I think this has come up a few times, in general we don't specify one way or another if `Timestep` is the same but there's a definite performance improvement on sharing the same one.